### PR TITLE
product-detail-page

### PR DESCRIPTION
**Pull request by chanchal-panpaliya**
product-detail-page added

### CODE REVIEW

**Review by sahiba0915 — COMMENTED** [review]
added each category very well, code looks good to me

File: src/my-web/pages/ProductDetails/ProductDetails.jsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+//sub files - components
+import TopOffer_ProductDetails from './SubPages/TopOffer';
+import Grocery_ProductDetails from './SubPages/Grosary';
+import Mobile_ProductDetails from './SubPages/Mobile';
+import Fashion_ProductDetails from './SubPages/Fashion';
+import Electronics_ProductDetails from './SubPages/Electronics';
+import Home_ProductDetails from './SubPages/Home';
+import Application_ProductDetails from './SubPages/Application';
+import BeautyToysMoreData_ProductDetails from './SubPages/BeautyToy';
+//css
```
Comment (Sharmaryan) [inline src/my-web/pages/ProductDetails/ProductDetails.jsx:21]: you can use a single file for import from subpages

File: src/my-web/pages/ProductDetails/ProductDetails.jsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+const ProductDetails = () =>{
+     const {getsingledata} = useContext(CartContext);
+     const {id} = useParams();
+     const [productData,SetProductData] = useState([]);
+
+    useEffect(()=>{
+       let selectedProduct = JSON.parse(localStorage.getItem('DATA_SELECTED'));
+        fetchProductDetailsData(id).then(function(result){
+            console.log(result)
```
Comment (Sharmaryan) [inline src/my-web/pages/ProductDetails/ProductDetails.jsx:33]: you can remove console.log

File: src/my-web/pages/ProductDetails/SubPages/Application.jsx [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+            <h3> Rating and Review </h3>
+            {
+             props.data.RatingReviewData.length>0 ? <div> {props.data.RatingReviewData.map((reviewitem,reviewindex)=>{
+                  return(
+                      <ul className='pd-padding-top-bottom' key={reviewindex}>
+                          <li> <b>  {reviewitem.title} </b> </li>
+                          <li> {reviewitem.desc} </li>
+                          <li> <Rating value={Number(reviewitem.rating)} reviewText={""} /> </li>
+                          <li> { reviewitem.images!=="" ? reviewitem.images.map((img,index)=>{
+                                 return <img src={img} className="pd-review-images"/>
```
Comment (Sharmaryan) [inline src/my-web/pages/ProductDetails/SubPages/Application.jsx:183]: you can take care of destructuring part

**Review by Sharmaryan — APPROVED** [review]
you can take care of destructuring part, also if you are importing from the same folder you can make use of barrel export,  other than that everything is fine

**Pull request merged**